add-ons/che/che.addon

```diff
@@ -0,0 +1,26 @@
+# Name: mini-che
+# Description: Creates a Eclipse Che in minishift
+# Required-Vars: 
+
+echo NOTE: To delete this addon, execute:
+echo oc delete project/mini-che
+
+echo Creating mini-che project
+oc new-project mini-che --description="Eclipse Che on minishift" --as=developer
+
+echo [CHE] Switching to mini-che...
+oc project mini-che
+
+echo [CHE] Setting admin role to che service account...
+oc apply -f che-admin-service-account.yml
+
+echo [CHE] Deploying Che on minishift (image docker.io/eclipse/che-server:nightly)
+oc new-app --param DOMAIN="#{ip}.nip.io" mini-che-template.yml -n mini-che 
+
+echo Please wait while the pods all startup!
+echo -
+echo You can watch in the OpenShift console via:
+echo   minishift console
+echo -
+echo Then you should be able the open the Che dashboard here:
+echo   http://che-mini-che.#{ip}.nip.io
```

Comment:
we can avoid line #11 to # 26 and just do
oc apply -f templates/che-single-user.json -n openshift